.pre-commit-config.yaml

```diff
@@ -1,37 +1,38 @@
 repos:
   - repo: https://github.com/python-poetry/poetry
-    rev: "fb493f2e80a11403fa289617a703ae622d4ccd75" # frozen: 1.3.0
+    rev: "7fe4678f92dcab48a35169db6567cbb3f5a7e3ef" # frozen: 1.4.0
     hooks:
       - id: poetry-check
       - id: poetry-lock
         args:
           - "--no-update"
 
   - repo: https://github.com/floatingpurr/sync_with_poetry
-    rev: 6fd6194c7bbd629b18a7bcd7a4a38a57f128b451 # 0.4.0
+    rev: 6fd6194c7bbd629b18a7bcd7a4a38a57f128b451 # frozen: 0.4.0
     hooks:
       - id: sync_with_poetry
+        pass_filenames: false
         files: ".pre-commit-config.yaml|poetry.lock"
-        args: ["--skip", "isort"]
+        args: ["poetry.lock"]
 
   - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+    rev: 5.11.5
```

Comment:
Since all other versions are frozen, we should probably also freeze this version? Also, the latest version is `5.12.0`, shouldn't we use the latest version?